package.json

```diff
@@ -105,17 +105,17 @@
     "@guardian/ab-core": "^2.0.0",
     "@guardian/ab-react": "^2.0.1",
     "@guardian/consent-management-platform": "^6.12.2",
-    "@guardian/libs": "^3.0.0",
-    "@guardian/src-accordion": "^3.8.0",
-    "@guardian/src-brand": "^3.8.0",
-    "@guardian/src-button": "^3.8.0",
-    "@guardian/src-checkbox": "^3.8.0",
-    "@guardian/src-foundations": "^3.8.0",
-    "@guardian/src-icons": "^3.8.0",
-    "@guardian/src-layout": "3.8.0",
-    "@guardian/src-link": "^3.8.0",
-    "@guardian/src-radio": "^3.8.0",
-    "@guardian/src-text-input": "^3.8.0",
+    "@guardian/libs": "^3.1.0",
```

Comment:
Nitpick but this lib isn't aprt of source

Reply:
all good if I add it to the description of the PR?